shared/engine/index.js

```diff
@@ -410,6 +414,7 @@ class Engine {
 
   reset () {
     resetClient()
+    this._setupClient()
```

Comment:
Let's just do a platform check for now so it only works on desktop

On Friday, August 12, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/engine/index.js
> <https://github.com/keybase/client/pull/3881#discussion_r74676146>:
>
> > @@ -410,6 +414,7 @@ class Engine {
> >
> >    reset () {
> >      resetClient()
> > +    this._setupClient()
>
> Doesn't look it works on mobile. Even just doing resetClient() breaks
> future rpc requests.
>
> I think we should hold off on mobile hmr engine until you finish your
> changes.
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3881/files/a8c953c911bcc1c0333582a7b29b88cc77da1dca#r74676146>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhUxKFJ2YRf-w7wn2wAxHX89iKDCjVks5qfQq8gaJpZM4Jjggf>
> .
>